### ISSUE TITLE
False positive E321 in JuMP macros

### ISSUE THREAD

**Issue by ufechner7** [tonyhffong/Lint.jl#199]
The third line of this code code is marked with E321: area use of undeclared symbol
```
@variable(m,  0.15 <= area   <= 2500.0)
@variable(m, 60.0 <= height <= 500.0)
@NLconstraint(m, force1(area, height) == force2(area, height))
```
even though these variables are declared before. How can I suppress these warnings?
